### PR TITLE
Register process

### PR DESCRIPTION
**Pull request by thejoefriel**
This relates to epic #23 

In this PR it now handles the entire user story registering on the dashboard. From /register, they go through the onboarding process with client side validation and then are stored in our database. 

NOTE: There are no tests on this currently 

## Checklist Criteria
- [x] **Branch is up to date with master**
- [x] **The project runs**
- - [x] You can download and install locally, and run the project locally
- - [x] All existing and new features run without bugs
- - [x] The build script runs without error
- - [ ] (If the project is already deployed to Heroku) the branch to be merged can deploy successfully to Heroku
- [x] **Code is consistent**
- - [x] Linter is used, no error messages
- - [x] Code is consistent with agreed team style
- - [x] Naming of files is consistent with project (e.g. all files named with camelCase)
- [ ] **Tests are in place**
- - [ ] Test file is in the same folder as the original file, with the extension .test.js
- - [ ] Each new file written has a test file (where possible)
- [ ] **Tests all pass**
- - [ ] All tests should pass locally and on Travis
- - [ ] Edge cases have been considered in tests
- [ ] **Test coverage is maintained**
- - [ ] Project coverage stays > 95%
- - [ ] PR will not bring down coverage by more than 1%
- - [ ] Files and lines of code have been ignored if they cannot be tested
- - [ ] If you are stuck with writing tests, you have assigned @arrested-developer to the PR and added a comment to explain what help is needed
- [x] **No console logs**
- - [x] Console.log is only used for crucial system messages (e.g. which port server is on).
- - [x] Any console.logs used for debugging have been removed.
- - [x] Console.error is used in error handling.
- - [ ] global.console.log and global.console.error are mocked in tests when needed to avoid littering the console

### CODE REVIEW

**Comment by thejoefriel** [comment]
Travis failing it due to eslint warnings - fixing now...

File: client/src/Components/Common/Questions/Questions.style.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  }
+  /* .helpertext {
+    opacity: 0.8;
+    size: 16px;
+  } */
+
+  /* ${({ unanswered }) => unanswered && "color: red;"} */
+`
+
+const TextField = styled(CommonStyles)`
```
Comment (ismail2009) [inline client/src/Components/Common/Questions/Questions.style.js]: @thejoefriel @dupreesi  don't forget to **remove commented** code if yu don't need it
